examples/example_modal_decomposition_data.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+from lasy.profiles.transverse.transverse_profile_from_data import (
+    TransverseProfileFromData,
+)
+from lasy.profiles.transverse.hermite_gaussian_profile import (
+    HermiteGaussianTransverseProfile,
+)
+from lasy.profiles.longitudinal.gaussian_profile import GaussianLongitudinalProfile
+from lasy.profiles.combined_profile import CombinedLongitudinalTransverseProfile
+from lasy.utils.mode_decomposition import hermite_gauss_decomposition
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import skimage
```

Comment:
Just a note: if we test this script in CI, we'll need to install a few more libraries. Maybe we should have an `examples/requirements.txt`? This is, however, not required for this PR.